demos/golang/grpc_pingpong/prepare_ping_pong.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+# sanity check
+FILE_SET="go.mod
+ping
+pong
+occlum_ping
+occlum_pong
+go.mod
+go.sum
+pingpong/pingpong.pb.go
+pingpong/pingpong_grpc.pb.go"
+for CURR_FILE in $FILE_SET
+do
+        if [ -f "$CURR_FILE" ]; then
+                rm $CURR_FILE
+        fi
+done
+
+DIR_SET="occlum_ping_instance
+occlum_pong_instance"
+for CURR_DIR in $DIR_SET
+do
+        if [ -d "$CURR_DIR" ]; then
+                rm -fr $CURR_DIR
+        fi
+done
+
+# enable Go modules for package management 
+export GO111MODULE=on
+
+# update PATH so that the protoc compiler can find the plugin:
+export PATH="$PATH:$(go env GOPATH)/bin"
+
+# assume that protoc is installed
+if ! type "protoc" > /dev/null; then
```

Comment:
It seems that both stdout and stderr of the `type` command should be suppressed to make both the success and failure cases transparent.  So the command should be
```bash
if ! type "protoc" > /dev/null 2>&1; then
```

Reply:
Done